src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy

```diff
@@ -99,6 +99,9 @@ class DirectSynchronousExecutionJobManager extends BatchEuphoriaJobManager<Direc
 
     @Override
     BEJobResult submitJob(BEJob job) {
+        if (forbidFurtherJobSubmission) {
+            throw new BEException("You are not allowed to submit further jobs. This happens, when you call waitForJobs().")
```

Comment:
Wondering: Is this not only true for **while** `waitForJobs()` is active? Should the job-manager not become open after the waiting again? I also see another (long term) solution, namely to wait for one set of jobs in one thread, but allow submission to a new, independent set of jobs in another. (not to implement now, but maybe to think about).